spec/models/comment_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'validations' do
+    it 'validates numericality of author_id' do
+      comment = Comment.new(author_id: 'not_an_integer')
+      expect(comment).not_to be_valid
+    end
+
+    it 'validates presence of text' do
+      comment = Comment.new(text: nil)
+      expect(comment).not_to be_valid
+    end
+  end
+
+  describe 'associations' do
+    it 'belongs to a post' do
+      association = described_class.reflect_on_association(:post)
+      expect(association.macro).to eq(:belongs_to)
+    end
+  end
+end
```

Comment:
- [ ] As requested by the previous reviewer, kindly ensure that all your tests are passing, currently you have 9 failures.
![image](https://github.com/AlyAtalla/blog-app/assets/81932478/99325826-400d-4177-8beb-ba03524987df)

There seems to be this recurring error for all failing tests, `ActiveRecord::`. Check the screenshots below.
![image](https://github.com/AlyAtalla/blog-app/assets/81932478/08f9c16f-239e-411c-9bcc-49f13be0ad2c)

![image](https://github.com/AlyAtalla/blog-app/assets/81932478/0d170d1c-d2da-4d4d-9bd0-57003fed42d1)